.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-close: 60
```

Comment:
I think we should have an explicit discussion about a reasonable time before closing, which should be based on _why_ exactly we want this bot to close stale issues/PRs.

(Not saying that 60 days is bad; just that it should be a reasoned choice rather than a "eh, sounds alright to me" default.)

Reply:
> which should be based on why exactly we want this bot to close stale issues/PRs.

To automate busywork that we're already doing: removing old issues/PRs that have hit a dead end where we need more information. This is just a programmatic way of doing that.

I chose 60 days because it sounded alright to me :). We could bump it up to 6 months to only remove super old stuff. I don't really know how to come up with a "reasonable" time honestly. Any ideas?

Reply:
If it's _only_ occasional spring cleaning rather than aggressively keeping issue count down, a longer period is indeed more reasonable. (I would have picked 30 days as "sounds good to me" default, myself.)

Rough guesstimate: anything that hasn't seen any action since the last release is probably dead -- either hasn't been reproduced, or needs so much rebasing that it's better to start from scratch.

Reply:
30 days or less is good. More than that means we spent time responding to an issue that the reporter has lost context on, and we also have to reload the context we just spent triaging it.

6 months is far too long.

Reply:
Then let's pick 30 days and see how that works.